### PR TITLE
Remove django-pipeline.

### LINKED ISSUE #177: Django-Pipeline needs to be removed.
This is an old dependency that we no longer need, but which still breaks the collectstatic command. It can be safely removed.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Fixes #177.

## Changes in this PR.

- [X] Remove django-pipeline.

## Testing this PR.

1. Run collectstatic.

### Expected Output.

No errors.

@osuosl/devs

### CODE REVIEW

**Comment by athai** [comment]
Lgtm! :1:

File: streamwebs_frontend/streamwebs_frontend/settings.py.dist [hunk 1]
```diff
@@ -169,29 +168,6 @@ SCHOOL_CHOICES = (
     ('a', 'School A'),
 )
 
-# Pipeline settings for static files
-# Used to package, compress, and minimize our bower dependencies (e.g. JQuery)
-# https://axiacore.com/blog/effective-dependency-management-django-using-bower/
-# http://django-pipeline.readthedocs.io/en/latest/installation.html
-
-STATICFILES_STORAGE = 'pipeline.storage.PipelineCachedStorage'
-STATICFILES_FINDERS = (
```
Comment (Kennric) [inline streamwebs_frontend/streamwebs_frontend/settings.py.dist]: Isn't this necessary for normal static file mgmt? I think just the pipeline line should be removed.

**Comment by LyonesGamer** [comment]
Yep. My mistake, I misread that.

**Comment by Kennric** [comment]
+1

**Pull request merged**